channels/al.m3u

```diff
@@ -11,6 +11,8 @@ http://us.bestvideostreaming.is/8737/playlist.m3u8
 https://shkoder.gjirafa.com/api/media/rgjirafa/t0110y/index.m3u8
 #EXTINF:-1 tvg-id="Cufo.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/cjVAXnZ.png" group-title="Kids",Çufo (576p)
 http://93.157.62.180/Cufo/index.m3u8
+#EXTINF:-1 tvg-id="EuronewsAlbania.al" tvg-country="AL" tvg-language="Albanian" tvg-logo="https://i.imgur.com/hTBah1z.jpg" group-title="News",Euronews Albania (1080p)
+https://iptv.dc-team.com/query?url=https://www.youtube.com/watch?v=Sw2XgYV8GbI
```

Comment:
That link will expire eventually. Use channel link + "/live"

Reply:
They stream 2 things and they keep the most recent stream to show on live. So it isn't easy...